repository/h.json

```diff
@@ -1538,6 +1538,19 @@
 				}
 			]
 		},
+		{
+			"name": "Hyperscript",
+			"details": "https://github.com/gnat/hyperscript-sublime",
+			"labels": ["language syntax", "syntax", "html", "script", "highlighting", "htmx"],
+			"author": "gnat",
+			"description": "Syntax highlighting for Hyperscript (companion of HTMX)",
+			"releases": [
+				{
+					"sublime_text": ">=4075",
```

Comment:
With stable releases being available those (e.g. 4107) should be the first supported ones. All older dev builds have not been broadly available and probably don't need to be supported.